src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -242,6 +242,9 @@ public static RequestExecutor CreateForSingleNodeWithoutConfigurationUpdates(str
 
                     await ExecuteAsync(node, null, context, command, shouldRetry: false).ConfigureAwait(false);
 
+                    if (command.Result.Etag == -1)
+                        throw new InvalidOperationException($"Uninitialized topology ({nameof(Topology.Etag)}: -1)");
```

Comment:
Can you suggest something? I was not sure what does it exactly mean that the topology has etag == -1.